### PR TITLE
change for checking out FOV of T1

### PR DESCRIPTION
**Pull request by anbai106**
It is my first time to do the pull request, So i am not sure If i do it correctly, i just follow the steps from nipype..
Thanks in advance

### CODE REVIEW

**Comment by oesteban** [comment]
I have quickly looked for the ```--cw256``` flag and did not see that it is specified in mm, therefore I guess it is specified in voxels. If the latter is the case, this PR would break the correct behavior of the interface.

Since I'm not an expert on freesurfer, please @satra, take a look.

File: nipype/workflows/smri/freesurfer/autorecon1.py [hunk 1]
```diff
@@ -26,9 +28,11 @@ def checkT1s(T1_files, cw256=False):
     origvol_names = ["{0:03d}.mgz".format(i + 1) for i in range(len(T1_files))]
 
     # check if cw256 is set to crop the images if size is larger than 256
-    if not cw256 and any(dim > 256 for dim in shape):
+    if not cw256 and (voxel_size[0] * t1_size[0] > 256) or (voxel_size[1] * t1_size[1] > 256) or (voxel_size[2] * t1_size[2] > 256):
         print("Setting MRI Convert to crop images to 256 FOV")
         cw256 = True
+    else:
+        print("No need to add -cw256 flag")
```
Comment (anbai106) [inline nipype/workflows/smri/freesurfer/autorecon1.py]: I have some data, which size of image is small than 256, but it still give me the error that we should use the cw256 flag, so I think the FOV is specified in mm, which corresponds to number of voxels multiply dimension of voxels. 
And for the nipype logging, I followed the site from nipype, and I know that in nipype, we can not use python logging module, which is a bug and you did not figure it out, and the nipyep logging [page()](http://nipype.readthedocs.io/en/latest/users/config_file.html) is not so clear to follow, so do you have some more elaborate page to follow???

File: nipype/workflows/smri/freesurfer/autorecon1.py [hunk 2]
```diff
@@ -26,9 +28,11 @@ def checkT1s(T1_files, cw256=False):
     origvol_names = ["{0:03d}.mgz".format(i + 1) for i in range(len(T1_files))]
 
     # check if cw256 is set to crop the images if size is larger than 256
-    if not cw256 and any(dim > 256 for dim in shape):
+    if not cw256 and (voxel_size[0] * t1_size[0] > 256) or (voxel_size[1] * t1_size[1] > 256) or (voxel_size[2] * t1_size[2] > 256):
```
Comment (satra) [inline nipype/workflows/smri/freesurfer/autorecon1.py]: @oesteban - that is correct.

File: nipype/workflows/smri/freesurfer/autorecon1.py (hunk 1 above)
Comment (satra) [inline nipype/workflows/smri/freesurfer/autorecon1.py]: was that data previously processed with `-cw256` flag? and would it be possible to put the `mri/orig/001.mgz` somewhere so we can check?

re: nipype logging - if there is no separate issue, could you please create one?

Comment (anbai106) [inline nipype/workflows/smri/freesurfer/autorecon1.py]: For my data, for the first time that I ran recon-all, i got error that said to use -cw256, so in my pipeline I added this flag, this is the 001.mgz for one of my subjects: [https://www.dropbox.com/s/xpmlzlffhekese0/001.mgz?dl=0](url)

Sorry, i dont get the question for nipype logging, you mean I create another bug for nipype and python logging?
Thanks in advance

Comment (anbai106) [inline nipype/workflows/smri/freesurfer/autorecon1.py]: Actually, For the logging bug, my colleague has posted this like 2 months ago, and you just closed this issue, here is the issue number [https://github.com/nipy/nipype/issues/1667](#1667)

**Comment by effigies** [comment]
This pull request is "orphaned," which means it has been deemed to be abandoned by its original author. Orphaned pull requests have not been rejected, and we hope that if a user sees one that will meet their needs with a little work, that they will fork it and open a new pull request (or, in the case of the original author, reopen the original PR).

We ask that all adopted PRs be updated to merge or rebase the current master. If you would like to adopt a PR and need help getting started, any of a number of contributors will be happy to help.

**Pull request closed without merging**